formal-spec/candidateproperties.tex

```diff
@@ -0,0 +1,153 @@
+\section{Candidate Properties}
+
+\subsection{Global Properties}
+
+\begin{property}[Software Update Proposal]\label{prop:submissions-for-all}
```

Comment:
This doesn't depend on the update mechanism, does it? The fact that events are immutable is a property of the underlying blockchain. In principle our ledger rules could be used to store voting events in an append only file system, or they could be used by a system that just discards the events. 

The other part of the property states that as a stakeholder you can submit a proposal. I guess this also depends on the properties of the blockchain, which should ensure that a submitted transaction will eventually added to it.